app/extensions/brave/locales/bn-IN/preferences.properties

```diff
@@ -26,7 +26,7 @@ paymentsSidebarText2=All transaction IP addresses are anonymized with technology
 paymentsSidebarText3=Brave Bitcoin Wallets are provided through a partnership with:
 paymentsSidebarText4=Your contributions in the form of credit cards and bank cards are handled by:
 accountBalance=account balance
-accountBalanceConnectionError=error, can't retrieve data
```

Comment:
For l10n updates pls only update en-US the other files will be pulled down from transifex.